js/test/components/home.test.tsx

```diff
@@ -16,4 +42,19 @@ describe("home", () => {
     );
     expect(view.getByText(/fit for duty/i)).toBeInTheDocument();
   });
+
+  test("can reload the List on new sign-in", async () => {
+    const view = render(
+      <MemoryRouter>
+        <Home />
+      </MemoryRouter>,
+    );
+    await userEvent.click(
+      view.getByRole("button", { name: "Sign In Operator" }),
+    );
+
+    (fetch as jest.MockedFunction<typeof fetch>).mockClear();
+    await userEvent.click(view.getByRole("button", { name: "Complete" }));
+    expect(fetch).toHaveBeenCalledWith("/api/signin?line=blue");
```

Comment:
**question:** just to clarify, is the point of the `mockClear` above so that we can be sure that this `expect` is catching the reload and not the fetch that happens on the initial render?

Reply:
Yes 👍 I'll note that in a comment

Reply:
56c3aef